### PR TITLE
DELETE + UPDATE

### PR DESCRIPTION
**Pull request by alapan-sau**
@NirmalManoj Check the pr. We already tested all delete and update functions!

### CODE REVIEW

**Review by pavani-17 — APPROVED** [review]
Looks fine

**Comment by NirmalManoj** [comment]
Checking.

File: app.py [hunk 1]
```diff
@@ -1018,7 +1018,7 @@ def showPageOptions():
         tmp = sp.call('clear', shell=True)
         refreshDatabase()
         print("Choose an option from below: ")
```
Comment (NirmalManoj) [inline app.py:1020]: If I understand correctly,  `refreshDatabase()` will be unnecessary for our CLI if we check for integrity while updating and deleting. It's there in insertions; if it's there in delete and update as well, we probably don't need this.

**Review by NirmalManoj — APPROVED** [review]
The code looks good to me.

File: app.py (hunk 1 above)
Comment (NirmalManoj) [reply app.py:1020]: We can think about it and change accordingly in another PR.

**Pull request merged**